src/components/tabs/tabs.scss

```diff
@@ -0,0 +1,49 @@
+@use "../../settings/constants";
+
+.mykn-tabs {
+  &__buttons {
+    background-color: var(--theme-shade-50);
+    border: 1px solid var(--theme-shade-100);
+    width: fit-content;
+    display: flex;
+    align-items: center;
+    gap: 17px;
+    padding: 6px 40px;
```

Comment:
Don't forget to set box-sizing explicitly when setting padding. Also: can you add width: 100% so the button area will by default cover the full width of the tab?